libr/include/r_search.h

```diff
@@ -34,8 +34,6 @@ typedef struct r_search_keyword_t {
 	ut8 *bin_binmask;
 	ut32 keyword_length;
 	ut32 binmask_length;
-	ut32 idx[R_SEARCH_DISTANCE_MAX]; // searching purposes
```

Comment:
`idx[R_SEARCH_DISTANCE_MAX]` gets in the way when I made the PR. It introduced another nested loop which I did not know how to put it back in the code after the refactoring as the loop order had been swapped.